.env.example

```diff
@@ -1 +1,3 @@
-CHARACTERAI_TOKEN=
\ No newline at end of file
+CHARACTERAI_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+TEXTGENUI_ENDPOINT=http://127.0.0.1:5000/v1/chat/completions
```

Comment:
no w in textgenui inconsistency i am going to open a new issue

Reply:
you should have left this before i merged

Reply:
i only checked the commits after you merged

Reply:
bad team-member

Reply:
:(